cleanlab_studio/internal/api/api.py

```diff
@@ -673,6 +684,41 @@ def enrichment_preview(
     return cast(JSONDict, res.json())
 
 
+def run(
+    api_key: str,
+    new_column_name: str,
+    project_id: str,
+    prompt: str,
+    constrain_outputs: Optional[List[str]] = None,
+    extraction_pattern: Optional[str] = None,
+    optimize_prompt: Optional[bool] = True,
+    quality_preset: Optional[TLMQualityPreset] = "medium",
```

Comment:
```suggestion
    quality_preset: TLMQualityPreset = "medium",
```

Reply:
Why are we making quality_preset not optional here if it is optional in `enrichment_preview()` method?